include/fastdds/dds/xtypes/dynamic_types/DynamicType.hpp

```diff
@@ -64,8 +64,8 @@ class RTPS_DllAPI DynamicType : public std::enable_shared_from_this<DynamicType>
      * @retval RETCODE_OK when the member was found.
      * @retval RETCODE_BAD_PARAMETER when the member doesn't exists.
```

Comment:
Typo. Please fix every occurrence.

```suggestion
     * @retval RETCODE_BAD_PARAMETER when the member doesn't exist.
```

Reply:
Done in c8b1745ba

Reply:
This typo occurs again in `get_member` documentation

Reply:
Done in c8f6dde98